acsweb/action.yml

```diff
@@ -0,0 +1,29 @@
+name: 'UCSD Personal Site Deploy'
+author: 'Open Source @ UCSD'
+description: 'Sync your personal website to your acsweb.ucsd.edu site'
+inputs:
+  username:
+    description: 'The UCSD username where the acsweb site is hosted'
+    required: true
+  password:
+    description: 'The password for the UCSD username of the acsweb site'
+    required: true
+  local_dir:
+    description: 'The directory of the repository that contains the files you want to deploy'
+    required: false
+    default: ""
+  remote_dir:
```

Comment:
maybe not have this? since if one action has remote_dir of `subdir/`, but another just does the root, then the 2nd will overwrite the 1st

Reply:
nah lets keep it, there could be a setup where someone doesnt have conflicting remote_dirs.